src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -240,6 +240,7 @@ private void createCommitStatus(Run<?, ?> build,
         Map<String, String> envVars = Ghprb.getEnvVars(build, listener);
 
         String sha1 = envVars.get("ghprbActualCommit");
+        String statusUrl = getDescriptor().getStatusUrlDefault(this);
```

Comment:
This appears to be overriding a class variable, `private final String statusUrl`.  I would like your local var to not conflict with other variables defined in the scope of the class.  Please change this behavior.